From 8cae3aa38de1d546f6deb5d07a20c8cf1d4229b1 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Fri, 14 Jul 2017 23:14:26 +0000
Subject: [PATCH] MANATEE-346 Ship both Postgres 9.2 and 9.6 in sdc-manatee
 images Reviewed By: Richard Bradley <richard.bradley@joyent.com> Reviewed By:
 Trent Mick <trentm@gmail.com> Approved By: Trent Mick <trentm@gmail.com>

---
 targets.json.in              | 11 +++--------
 tools/prep_dataset_in_jpc.sh |  4 ----
 2 files changed, 3 insertions(+), 12 deletions(-)

diff --git a/targets.json.in b/targets.json.in
index 1196805..f12e7e5 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -977,15 +977,10 @@ cat <<EOF
     "build_platform": "20151126T062538Z",
     "image_name": "sdc-postgres",
     "image_description": "SDC manatee",
-    "image_uuid": "b4bdc598-8939-11e3-bea4-8341f6861379",
+    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
-        "postgresql92-server-9.2.4nb1",
-        "postgresql92-adminpack",
-        "postgresql92-datatypes",
-        "postgresql92-pgcrypto",
-        "postgresql92-replicationtools",
-        "postgresql92-upgrade",
-        "lz4-120"
+        "lz4-131nb1"
     ],
     "repos": [
       {"url": "git@github.com:joyent/sdc-manatee.git"}
diff --git a/tools/prep_dataset_in_jpc.sh b/tools/prep_dataset_in_jpc.sh
index 5f67460..a301bd8 100755
--- a/tools/prep_dataset_in_jpc.sh
+++ b/tools/prep_dataset_in_jpc.sh
@@ -317,10 +317,6 @@ if [[ -n "${packages}" ]]; then
   ${SSH} "/opt/local/bin/pkgin -f -y update"
   ${SSH} "touch /opt/local/.dlj_license_accepted"
 
-  if [[ ${build_name} == "manatee" || ${build_name} == "sdc-manatee" ]]; then
-    ${SSH} "/opt/local/bin/pkgin -y remove libuuid"
-  fi
-
   #
   # When pkgin fails, it ridiculously tells you that there are errors, but not
   # what the errors are (those are hidden in a log file). Since we destroy the
-- 
2.21.0

